ocean-client/src/vault-maxi.ts

```diff
@@ -25,7 +25,7 @@ class maxiEvent {
 
 export async function main(event: maxiEvent): Promise<Object> {
     let settings = await new Store().fetchSettings()
-
+    console.log("vault maxi v1.0-beta.1")
```

Comment:
Good idea to log a version. Should we store this somewhere else too, or is it enough by just putting it out to console? Maybe in checkSetup too? Or maybe it will make sense in future to trigger a debug / support identification that we can easier debug or check issues for other users, by getting anonymised data/information

Reply:
with this its shown on every invocation. was also thinking to maybe add it to the telegram prefix 
like [VaultMaxiv1.0 ...] ....

Reply:
Yes - on the other hand we shouldn't make it the prefix too long
And yes an own event for users could be too much to ask for, they will most likely just get telegram messages and based on those information we need to check, if something is not working as intended